cloud/deployment/inspect/inspect.go

```diff
@@ -246,6 +245,10 @@ func getDeploymentConfig(coreDeploymentPointer *astroplatformcore.Deployment, pl
 	if coreDeployment.SchedulerAu != nil {
 		deploymentMap["scheduler_au"] = *coreDeployment.SchedulerAu
 	}
+	if coreDeployment.CloudProvider != nil {
```

Comment:
Do we need to change condition on line 232 for region

Reply:
yes I think you are right